.yamato/config/__shared.metafile

```diff
@@ -83,11 +90,18 @@ test_platforms:
       - [Android_OpenGLES3, Android_Vulkan]: --timeout=2700
       - [iPhone]: --timeout=1800
       
-      - [Linux_OpenGlCore]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxOpenGlCoreLinear"
-      - [Linux_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxVulkanLinear"
-      - [Win_DX11]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX11Linear"
-      - [Win_DX12]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX12Linear"
-      - [Win_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsVulkanLinear"
+      #- [Linux_OpenGlCore]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxOpenGlCoreLinear"
+      #- [Linux_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxVulkanLinear"
+      #- [Win_DX11]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX11Linear"
+      #- [Win_DX12]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX12Linear"
+      #- [Win_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsVulkanLinear"
+
+      ### The method and values live in /com.unity.testframework.graphics/.../Editor/SetupProject.cs
+      #- [Win_DX11]: --extra-editor-arg="-executemethod" --extra-editor-arg="SetupProject.ApplySettings" --extra-editor-arg="d3d11"
```

Comment:
if we have all of these flags in here, and they apply for all standalones, could we uncomment them and thus not have them specified explicitly in each project file?

Reply:
the ones with CustomBuild dont apply anymore because of the ApplySettings, but the ApplySettings methods dont work for performance (intentionally) so i moved ApplySettings commands into the per-project metafiles

Reply:
I understand the situation for the new ApplySettings ones, but do we add a lot of value in keeping the CustomBuild ones commented out instead of simply deleted? I know I will be confused in a few months when I don't remember what they are for ;)

Reply:
Ohh I see, then indeed. Im wondering, the method that parses utr flags should usually overwrite if a flag is specified double, but I don't think it works on specifying multiple flags at once and forcing the order. But,  I assume we can only have one `--extra-editor-arg="-executemethod"` followed by something, so maybe we can extend it to consider also this case? I think its just another condition in here https://github.com/Unity-Technologies/Graphics/blob/master/.yamato/ruamel/jobs/shared/utr_utils.py#L18
What do you think?

Reply:
@theo-pnv i have deleted them in my local repo, just havent pushed yet :)

@kivistikliis thank you! i'll look now